### ISSUE TITLE
Air Assist activation - Sculpfun S30

### ISSUE THREAD

**Issue by mauroproductions** [arkypita/LaserGRBL#2023]
### Discussed in https://github.com/arkypita/LaserGRBL/discussions/1994

<div type='discussions-op-text'>

<sup>Originally posted by **Biggy159** January  9, 2023</sup>
Hello, I just bought Sculpfun S30 PRO MAX and there is only software activation of the air pump. I have to probably add button in LaserGRBL but I really do not know parameters which should I insert. Does anyone know how to solve this?

Thank you very much.</div>

**Comment by lsces** [comment]
Yes I know this was posted nearly 3 years ago, but I've only just got around to getting the Sculpfun S30 PRO setup to add to my collection of CNC kit.
I am fairly sure that I need M8 and M9 to toggle the air assist, and have set up a custom button, but I and getting 'unsupported function' in the left hand window when I use it :( Searching seems to confirm it should be M8 and M9, and I am fairly sure I tested it a while back but on the trial LightBurn Software. Anybody got any thoughts on where I am going wrong?

**Comment by lsces** [comment]
Initial problem solved ... Senior moment as I'd put G8&G9 in the button code. Next question is how to populate the icon on the button. I've found the mdiFan but not how to display it ...
In addition, while I can produce drawings for separate air on and air off layers, the obvious next step is to be able to mimic the Lightburn switch on a layer. 
Still I'm a hell of a lot further forward than I was 24 hours ago ... just need to move back of the old W10 box and back the the Linux desktop!
